loadgen/kitchensink/helpers.go

```diff
@@ -0,0 +1,139 @@
+package kitchensink
+
+import (
+	"context"
+	"fmt"
+	"github.com/golang/protobuf/proto"
+	"go.temporal.io/api/common/v1"
+	"go.temporal.io/sdk/client"
+	"golang.org/x/sync/errgroup"
+	"google.golang.org/protobuf/types/known/durationpb"
+	"time"
+)
+
+// Must match string used in rust generator
+const nonexistentHandler = "nonexistent handler on purpose"
+
+func NoOpSingleActivityActionSet() *ActionSet {
+	return &ActionSet{
+		Actions: []*Action{
+			{
+				Variant: &Action_ExecActivity{
+					ExecActivity: &ExecuteActivityAction{
+						ActivityType:        "noop",
+						StartToCloseTimeout: &durationpb.Duration{Seconds: 5},
+					},
+				},
+			},
+			{
+				Variant: &Action_ReturnResult{
+					ReturnResult: &ReturnResultAction{
+						ReturnThis: &common.Payload{},
+					},
+				},
+			},
+		},
+	}
+}
+
+type ClientActionsExecutor struct {
+	Client     client.Client
+	WorkflowID string
+	RunID      string
+}
+
+func (e *ClientActionsExecutor) ExecuteClientSequence(ctx context.Context, clientSeq *ClientSequence) error {
+	println("!!!!!!!! Number of action sets ", len(clientSeq.ActionSets))
```

Comment:
Do we need these `println` here still? Can we convert them to proper them to use the same logger the rest of the Go code uses?

Reply:
I'll just get rid of these, had forgotten to clean them up